backend/app/services/implementations/log_records_service.py

```diff
@@ -144,7 +144,7 @@ def get_log_records(
             logs.attn_to,\n \
             logs.note,\n \
             logs.tags,\n \
-            logs.building,\n \
+            logs.building_id,\n \
```

Comment:
you need to use the building name from the table
```suggestion
            buildings.name AS building_name,\n \
```

Reply:
and then at the bottom of the SQL query (where the joins are), you need to join the buildings table to this table like so:
```
JOIN buildings on logs.building_id = buildings.id
```